### PR TITLE
hook up handle click to individual cards, break card function into it…

### PR DESCRIPTION
**Pull request by Obleo33**
…s own file

### CODE REVIEW

File: src/DistrictCards/DistrictCards.css [hunk 1]
```diff
@@ -1,3 +1,11 @@
+
```
Comment (Obleo33) [inline src/DistrictCards/DistrictCards.css:1]: new color palate

File: src/DistrictCards/DistrictCards.js [hunk 2]
```diff
@@ -1,22 +1,14 @@
 import React from 'react';
-import DistrictYears from './DistrictYears';
 import './DistrictCards.css';
+import CardCreator from './CardCreator'
```
Comment (Obleo33) [inline src/DistrictCards/DistrictCards.js:3]: we should probably move this into its own folder

File: src/DistrictCards/DistrictCards.css (hunk 1 above)
Comment (becs919) [reply src/DistrictCards/DistrictCards.css:1]: love it! ^^^^

File: src/DistrictCards/DistrictCards.js (hunk 2 above)
Comment (becs919) [reply src/DistrictCards/DistrictCards.js:3]: agreed, as well as districtYearly!

**Comment by becs919** [comment]
love the new color pallet. Also agree with breaking out the two files into folders.

**Pull request merged**